### ISSUE TITLE
Clean up metamodels in EAP

### ISSUE THREAD

**Issue by anthonyanjorin** [eMoflon/emoflon#610]
Currently the TGG metamodel contains much more than is necessary in the EAP file.   I guess we should/could reduce this by removing everything that is not necessary (compiler etc).

**Comment by erhanleblebici** [comment]
One question related to this: My ultimate aim is to integrate MOSL into TGGs.eap anyway when everything is finished. Do you agree with this structure? If so, the EAP you mean here won't exist in the future. 
Regardless of this, it makes of course sense to remove unnecessary packages, at least for Thomas.

**Comment by anthonyanjorin** [comment]
Yes please...  the less EAPs the better :) You already know how I feel about this right...

**Comment by RolandKluge** [comment]
Is this task complete?
